src/azure-cli-core/azure/cli/core/adal_authentication.py

```diff
@@ -35,7 +35,7 @@ def _get_token(self, sdk_resource=None):
         """
         external_tenant_tokens = None
         try:
-            scheme, token, full_token = self._token_retriever(sdk_resource)
+            scheme, token, token_entry = self._token_retriever(sdk_resource)
```

Comment:
Is it just a rename operation? Does the object structure change?

Reply:
Yes. No.